src/main/kotlin/app/batch/S3Reader.kt

```diff
@@ -54,43 +53,54 @@ class S3Reader(private val s3client: AmazonS3, private val keyPairGenerator: Key
     @Synchronized
     private fun getS3ObjectSummariesIterator(s3Client: AmazonS3, bucketName: String): ListIterator<S3ObjectSummaryPair> {
         if (null == iterator) {
-            val request = ListObjectsV2Request().apply {
-                withBucketName(bucketName)
-                withPrefix(s3PrefixFolder)
+            val allSuffixes = s3SuffixesCsv.trim().split(",")
```

Comment:
This won't work if the s3SuffixesCsv is not set will it? Need to allow for no suffix as now so it just gets the ones in the s3PrefixFolder then.

Reply:
It's required anyway :-) Not optional :-)